host-go/config/config.go

```diff
@@ -30,20 +31,73 @@ func LoadFromFile[TSource any, TResult any](path string, src enumerable.Enumerab
 //
 // It does not enumerate the src.
 func Load[TSource any, TResult any](lensConfig model.Lens, src enumerable.Enumerable[TSource]) (enumerable.Enumerable[TResult], error) {
-	modules := []module.Module{}
-	for _, lensModule := range lensConfig.Lenses {
-		var module module.Module
+	runtime := wasmer.New()
```

Comment:
thought: This function doesn't seem to be used anywhere at the moment but it seems inefficient to create a new runtime on load. The fact that it's hiding that memory heavy action could cause unexpected consequences for users. Shouldn't we force users to provide the runtime? We could remove this function and let users use `LoadInto` and `LoadIntoFromFile`.

Reply:
This is a simple interface/func that allows users to not worry about a large amount of the complexities of lens.  Here they can just provide a cfg and a src, and not worry about anything.

Not everyone will care about construction-performance.